ui/src/services/NodeVolumesUtils.js

```diff
@@ -280,6 +281,6 @@ export const getVolumeListData = createSelector(
       weights[STATUS_HEALTH] = 0;
 
       return weights[b.health] - weights[a.health];
-    })
+    });
```

Comment:
I would also extract the ranking logic outside of this function. The responsibility of this function is only to retrieve the data for VolumeTable.

But you can address it ticket #2876.